### PR TITLE
Added chapter for default-snippets and custom-urls

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: cookbook/custom-urls.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
+How to manage custom-urls?
+==========================
+
+Sulu gives the content-manger an easy way to manage custom-urls or landing
+pages. This urls can be defined in the webspace-settings. A custom-url is
```
Comment (danrot) [inline cookbook/custom-urls.rst]: `These urls`

File: cookbook/custom-urls.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+basically an additional url for a specific page.
```
Comment (danrot) [inline cookbook/custom-urls.rst]: I would also add somehow that the localization is already included (without the localization specific part int he URL)

File: cookbook/custom-urls.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Description
+    * - Title
+      - This title will be displayed in the list and should identify the usecase
+        of this url.
+    * - Activated
+      - This flag indicates the state of the custom-url.
```
Comment (danrot) [inline cookbook/custom-urls.rst]: `indicates if the custom-url is shown on the website`

File: cookbook/custom-urls.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+.. note::
+
+    If the target page gets deleted the custom-url will stay in the list, will
+    be marked and returns a 404 on the website.
+
+How to configure custom-urls?
+-----------------------------
+
+The custom-urls can be configured in the ``webspace.xml`` in the environment.
```
Comment (danrot) [inline cookbook/custom-urls.rst]: I would explain that these values have impact on the `Base Domain` dropdown mentioned previously.

File: cookbook/custom-urls.rst [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        indexed by search engines.
+    * - No Index
+      - The page will not be indexed by search engines.
+    * - No Follow
+      - The links on the page will not be followed by search engines.
+
+.. note::
+
+    If the target page gets deleted the custom-url will stay in the list, will
+    be marked and returns a 404 on the website.
```
Comment (danrot) [inline cookbook/custom-urls.rst]: Marked as what? Invalid?

File: cookbook/custom-urls.rst [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+The custom-urls can be configured in the ``webspace.xml`` in the environment.
+
+.. code-block:: xml
+
+    <webspace>
+        ...
+
+        <portals>
+            ...
```
Comment (danrot) [inline cookbook/custom-urls.rst]: Remove, there won't be any lines in between (at least not in an easy setup)

File: cookbook/custom-urls.rst [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+                            <custom-url>sulu.lo/*</custom-url>
+                            <custom-url>*.sulu.lo</custom-url>
+                        </custom-urls>
+                    </environment>
+                </environments>
+            </portal>
+        </portals>
+    </webspace>
+
+It is possible to set to "wildcard" ``*`` for subdomains or folders.
```
Comment (danrot) [inline cookbook/custom-urls.rst]: `It is possible to set a wildcard (``*``) for subdomains and folders, which can be replaced by arbitrary values when creating a custom-url.`

File: cookbook/default-snippets.rst [hunk 8]
```diff
@@ -0,0 +1,32 @@
+How to define a default-snippet?
+================================
+
+Sulu gives the content-manager an easy way to define default snippets per type.
+This default-snippets automatically be used if the snippet-selection is empty on
```
Comment (danrot) [inline cookbook/default-snippets.rst]: `These default-snippets ...`

File: cookbook/default-snippets.rst [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+a specific page. Be aware of that this "default" selection will not be displayed
```
Comment (danrot) [inline cookbook/default-snippets.rst]: `~~Be aware of that~~ This default selection will...` Don't make sentences more complicates than they need to be.

File: cookbook/default-snippets.rst [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+in the form and can have a big impact on the page.
+
+.. note::
+
+    This feature is disabled by default and can be activated by setting the
+    config value `sulu_snippet.types.snippet.fallback_enabled` to true.
+
+When the defaults will be used?
```
Comment (danrot) [inline cookbook/default-snippets.rst]: `When will the defaults be used?`

File: cookbook/default-snippets.rst [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+-------------------------------
+
+There have to be conditions which have to match:
```
Comment (danrot) [inline cookbook/default-snippets.rst]: The default snippets will be used if the following conditions are fulfilled.

File: cookbook/default-snippets.rst [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+1. The feature has to be activated
+2. The snippet-selection parameter ``snippetType`` and ``default`` has to be
```
Comment (danrot) [inline cookbook/default-snippets.rst]: Is it ``default`` or ``fallback``? Your link in the next line says ``fallback``.

File: cookbook/default-snippets.rst [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+   defined. See :doc:`../reference/content-types/snippet`.
+3. The default snippet for the configured type has to be selected in the
+   webspace settings.
+
+If this conditions match the default snippet will be injected to the page. The
```
Comment (danrot) [inline cookbook/default-snippets.rst]: `these conditions`

File: cookbook/default-snippets.rst [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+template of the page does not have to be changed.
```
Comment (danrot) [inline cookbook/default-snippets.rst]: Does this last sentence make any sense? Could anybody have the idea that the template should be changed? I think this sentence confuses more than it helps 😕

File: cookbook/default-snippets.rst [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+Can i use the default snippet without the content-type?
```
Comment (danrot) [inline cookbook/default-snippets.rst]: `Can I use the default snippets without the content type?`

File: cookbook/default-snippets.rst (hunk 12 above)
Comment (danrot) [inline cookbook/default-snippets.rst]: Forget about it, have seen that you've also updated it in this PR.

File: reference/twig-extensions/functions/sulu_snippet_load_default.rst [hunk 16]
```diff
@@ -3,6 +3,15 @@
...
 Returns content array of selected default snippet for given snippet type.
 
+.. code-block:: jinja
+
+    {% set snippets = sulu_snippet_load_default('default') %}
+    {{ snippets[0].title }}
+
+.. note::
+
+    The system currently only supports on default per type.
```
Comment (danrot) [inline reference/twig-extensions/functions/sulu_snippet_load_default.rst]: `support one default`

**Comment by danrot** [comment]
General question: since when have we decided to use questions as headlines? Can't find to many occurrences of that, and I think we should be consistent if possible. And personally I don't like questions as headlines...

/cc @chirimoya @webmozart

File: cookbook/default-snippets.rst (hunk 14 above)
Comment (wachterjohannes) [inline cookbook/default-snippets.rst]: i thought the twig-template should i change it to "twig-template" or remove it?

**Pull request merged**